### PR TITLE
clarify tests requirements closes #19261

### LINKED ISSUE #19261: clarify petsc options requirements in tests
## Reason
The current tests requirements were unclear and pointed out as an issue in the Grizzly sqa report

## Design
Work with Fande to update this.

### PR DESCRIPTION
**Pull request by lynnmunday**
closes #19261

### CODE REVIEW

**Comment by lynnmunday** [comment]
@fdkong will you review

**Review by fdkong — DISMISSED** [review]

**Comment by YaqiWang** [comment]
What is that `iThe`?

File: test/tests/misc/petsc_option_left/tests [hunk 1]
```diff
@@ -6,7 +6,6 @@
     type = RunApp
     input = '2d_diffusion_petsc_option.i'
     absent_out = "Option left.*value.*hypre"
-
-    requirement = "PETSc option database shall work properly"
+    requirement = "iThe system shall use the default PETSc option database in the master app to handle system-level PETSc parameters, such as '-log_view.' "
```
Comment (loganharbour) [inline test/tests/misc/petsc_option_left/tests:9]:
```suggestion
    requirement = "The system shall use the default PETSc option database in the master app to handle system-level PETSc parameters"
```

Extra detail is unnecessary for requirements. Also removes the extraneous i

**Review by loganharbour — CHANGES_REQUESTED** [review]

**Comment by lynnmunday** [comment]
thanks, I used @loganharbour commit.

**Review by loganharbour — APPROVED** [review]

**Pull request merged**